http.py

```diff
@@ -131,7 +133,8 @@ def send_request(self):
             self.sock.send(self.request)
         except socket.error as exc:
             print exc
-        self.get_response()
+        finally:
```

Comment:
whoa, thats not right. @csanders-git do you think self.get_response should be after `self.sock.send(self.request)`? Then 135 should have a TestError be thrown